spec/requests/requests_spec.rb

```diff
@@ -3,85 +3,243 @@
 require 'rails_helper'
 require 'telegram/bot/rspec/integration/rails'
 
-RSpec.describe 'Requests', telegram_bot: :rails do
+RSpec.describe 'Requests', type: :request do
   let(:organization) { create(:organization) }
 
-  describe 'POST /{organization_id}/requests' do
-    before(:each) { allow(Request).to receive(:broadcast!).and_call_original } # is stubbed for every other test
-    subject { -> { post organization_requests_path(organization, as: user), params: params } }
-    let(:params) { { request: { title: 'Example Question', text: 'How do you do?', hints: ['confidential'] } } }
-    let(:user) { create(:user, organizations: [organization]) }
+  describe 'GET /{organization_id}/requests' do
```

Comment:
```suggestion
  describe 'GET /:organization_id/requests' do
```

Reply:
You added this code in https://github.com/tactilenews/100eyes/pull/1972/commits/9783770b417694d6fcd0f1f9f37c0f606edea729#diff-71c5d728e921a45b367bd360e1bbeb7333547d2b23b5f1105ca6b72a941e3ec2R9

I favor using the rails annotation for these, which is why I've done that for the description I added. I don't think it's productive to try to force my styling on others. If you have changed your mind and prefer the annotation I added over the one you added, please feel free to update it going forward, or retroactively.

Reply:
I just wanted to have it consistently and are willing to go either way. Just found to formats in the file and proposing the style that i found simpler to type at that moment